Gopkg.toml

```
@@ -18,6 +18,10 @@
   name = "github.com/hashicorp/vault"
   version = "v1.0.0-rc1"
 
+[[constraint]]
+  name = "github.com/go-kit/kit"
+  version = "0.5.0"
```

there is a new version
```suggestion
  version = "v0.8.0"
```

---

I tried to change this but it causes conflicts with webpa-common, as that still has version 0.5.0 in it.  This can't be changed until it is changed in webpa-common.